ui/src/app/views/new-dataset-modal.tsx

```diff
@@ -227,7 +257,8 @@ class NewDataSetModal extends React.Component<Props, State> {
         <div style={{display: 'flex', alignItems: 'center', marginTop: '1rem'}}>
           <CheckBox style={{height: 17, width: 17}}
                     data-test-id='export-to-notebook'
-                    onChange={() => this.changeExportToNotebook()} />
+                    onChange={() => this.changeExportToNotebook()}
+                    checked={this.state.exportToNotebook} />
           <div style={{marginLeft: '.5rem',
             color: colors.black[0]}}>Export to notebook</div>
```

Comment:
probably not in the scope of this ticket, but it would be nice if this text `Export to notebook` could be clicked to check the checkbox. Currently, clicking the text does nothing.

Reply:
We can talk to lou/team but if we do agree to do this it should be consistent throughout the site